osgp/platform/osgp-core/src/main/resources/db/migration/V20201020120000000__Add_not_null_constraints_to_device_authorization.sql

```diff
@@ -0,0 +1,4 @@
+DELETE FROM device_authorization WHERE device IS NULL OR organisation IS NULL;
```

Comment:
Okay, als dit met Gerbert gecheckt is

Reply:
Yes, dit is gechecked met Gerbert en Maastricht